src/client/reducers/itemReducer.js

```diff
@@ -0,0 +1,25 @@
+import { ADD_ITEM, ADD_ITEM_SUCCESS } from '../components/SearchBar/actions';
+
+const addItem = (state = {}, action) => {
+  switch (action.type) {
+    case ADD_ITEM:
```

Comment:
you could use ADD_ITEM_SUCCESS here

Reply:
For now itemReducer is deleted - no need for this reducer